tutorials/admin/services/services.md

```diff
@@ -1 +1,75 @@
-# services
+# Services
+
+The Marble software can be configured to provide several services to the users of your node. Services are user-facing
+tools that largely fall under the following categories:
+
+- serving and cataloging data hosted on your server
+- interactive development environment
+- remote processing tools
+
+(enable)=
+## Enable an included service
+
+You can pick and choose which services you would like to offer your users by enabling them via the `env.local` file.
+
+The Marble software comes with several services already included. The list of services already included can be found
+in the 
+[Birdhouse documentation](https://birdhouse-deploy.readthedocs.io/en/latest/birdhouse/components/README.html).
+
+This documentation explains what each service is used for and how to enable and configure it.
+
+In general, the process to enable a service is a follows:
+
+1. Take note of the directory that contains the service's code. In general this will be on of the directories in the
+   `birdhouse/components` folder.
+2. Edit the `EXTRA_CONF_DIRS` variable in the `env.local` file to include the service's directory. This is a relative 
+   path from the `birdhouse/` directory *or* an absolute path. For example, to enable the Thredds service add the
+   string `./components/thredds` to the `EXTRA_CONF_DIRS` variable.
+3. Add or update any additional variables in `env.local` as required by the specific service (see the 
+   [Birdhouse documentation](https://birdhouse-deploy.readthedocs.io/en/latest/birdhouse/components/README.html)
+   for customization options for each service).
+4. Restart the software by running:
+
+   ```shell
+   ./pavics-compose.sh down
+   ./pavice-compose.sh up -d
+   ```
+
+   in order for the changes to take effect.
+5. Log in to Magpie as an admin user to [set the access permissions for the new service](../users/permissions.md)
```

Comment:
Let's link to Magpie documentation. By the way, where would Magpie be introduced in the admin side tutorials? Are we planning on writing a brief introduction somewhere about what Magpie does within Birdhouse?

Reply:
Yeah, it's introduced in that file that's linked (admin/users/permissions.md). See #33